projects/demo-cypress/src/tests/static-request.cy.ts

```diff
@@ -1,5 +1,5 @@
 import {TestBed} from '@angular/core/testing';
-import {TuiStaticRequestService} from '@taiga-ui/cdk';
+import {TuiStaticRequestService} from '@taiga-ui/legacy';
```

Comment:
Add schematic migration
https://github.com/taiga-family/taiga-ui/blob/main/projects/cdk/schematics/ng-update/v4/steps/constants/identifiers-to-replace.ts

Reply:
fixed